python/ray/tune/ray_trial_executor.py

```diff
@@ -541,8 +563,13 @@ def restore(self, trial, checkpoint=None):
                 assert type(value) != Checkpoint, type(value)
                 trial.runner.restore_from_object.remote(value)
             else:
-                worker_ip = ray.get(trial.runner.current_ip.remote())
-                trial.sync_logger_to_new_location(worker_ip)
+                # This can be very slow - a better fix would
```

Comment:
Maybe add a TODO here and clarification when this would be very slow? Also might want to create an issue if this can cause problems.